test/python/WMCore_t/ResourceControl_t/ResourceControl_t.py

```diff
@@ -5,39 +5,37 @@
 Unit tests for ResourceControl.
 """
 
-import unittest
-import threading
 import os
-import sys
 import subprocess
+import sys
+import threading
+import unittest
 
+from WMCore.Agent.Configuration import Configuration
+from WMCore.BossAir.RunJob import RunJob
+from WMCore.DAOFactory import DAOFactory
+from WMCore.FwkJobReport.Report import Report
+from WMCore.ResourceControl.ResourceControl import ResourceControl
+from WMCore.Services.UUID import makeUUID
 from WMCore.WMBS.File import File
 from WMCore.WMBS.Fileset import Fileset
 from WMCore.WMBS.Job import Job
 from WMCore.WMBS.JobGroup import JobGroup
-from WMCore.WMBS.Workflow import Workflow
 from WMCore.WMBS.Subscription import Subscription
-
-from WMCore.ResourceControl.ResourceControl import ResourceControl
-from WMQuality.TestInit import TestInit
-from WMCore.Services.UUID import makeUUID
-from WMCore.DAOFactory import DAOFactory
-from WMCore.Agent.Configuration import Configuration
-from WMCore.BossAir.RunJob import RunJob
+from WMCore.WMBS.Workflow import Workflow
 from WMCore.WMBase import getTestBase
-from WMCore.FwkJobReport.Report import Report
-from WMCore.Services.EmulatorSwitch import EmulatorHelper
+from WMQuality.Emulators.EmulatedUnitTestCase import EmulatedUnitTestCase
+from WMQuality.TestInit import TestInit
+
```

Comment:
This is all just cleaning up and reordering imports. The only change is the drop of EmulatorHelper